Makefile

```diff
@@ -130,7 +130,7 @@ rebuild_client_start: docker_check ## Rebuild and run a client daemon which is o
 
 .PHONY: client_connect
 client_connect: docker_check ## Connect to the running client debugging daemon
-	docker exec -it client /bin/bash -c "go run -tags=debug app/client/*.go debug"
+	docker exec -it client /bin/bash -c "POCKET_P2P_IS_CLIENT_ONLY=true go run -tags=debug app/client/*.go debug"
```

Comment:
In order to leverage `viper` it has to start with `POCKET_P2P` so how about `POCKET_P2P_CLIENT_DEBUG_MODE` ?

Otherwise I can put it at the root of the config and be `POCKET_CLIENT_DEBUG_MODE` (probably better if we think it can be useful for other purposes, I can't think about anything else currently

Reply:
I have tackled this in the other PR here https://github.com/pokt-network/pocket/pull/425/commits/d313d978db477c496dc67403b600b2a6cdd7ed5e because the implementation was just easier there (due to the improvements to module initialization) and doing it here would have required some extra work that than I would have had to refactor anyway.